apps/dot_ext/models.py

```diff
@@ -30,7 +30,7 @@ class Application(AbstractApplication):
     op_tos_uri = models.CharField(default=settings.TOS_URI, blank=True, max_length=512)
     op_policy_uri = models.CharField(default="", blank=True, max_length=512)
     client_uri = models.CharField(default="", blank=True, max_length=512, verbose_name="Client URI",
```

Comment:
if you update this to be a [URLField](https://docs.djangoproject.com/en/2.1/ref/models/fields/#urlfield) you can do without the extra logic in the form.

Reply:
This sounds great for taking advantage of the URLField type field!